.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: python
 python:
+  - "2.7"
   - "3.4"
 install:
     - pip install -r requirements.txt
     - pip install braintree coveralls mock mysqlclient
 env:
-  - DJANGO=1.9
   - DB=postgres
 addons:
   postgresql: "9.4"
 script:
   - coverage run --omit=*/tests/* --source=hc manage.py test
-notifications:
```

Comment:
Any particular reason this file was changed in this PR?

Reply:
The yml syntax was having a problem, which caused our pushes not to start a build so we reverted to not using on_failure. So I deleted and put it at the bottom. Solved the problem. I think it must have been the indentations